js/main.js

```diff
@@ -0,0 +1,164 @@
+};
+const speaker5 = {
+  imageSrc:
+    'https://res.cloudinary.com/optprime/image/upload/v1646837245/module1-capstone/lewis_dwouxe.jpg',
+  speakerName: 'Sir Lewis Hamilton',
+  title: 'Seven-time world F1 champion ',
+  speakerDescription:
+    'Lewis Hamilton has defended his interest in raising awareness about environmental issues, insisting his Mercedes team is working to become "carbon neutral."',
+};
+const speaker6 = {
+  imageSrc:
+    'https://res.cloudinary.com/optprime/image/upload/v1646837318/module1-capstone/sebastian-vettel_ch6odf.jpg',
+  speakerName: 'Sebastian Vettel',
+  title: 'F1 Driver for Aston Martin',
+  speakerDescription:
+    'He has been shining a spotlight on sustainability and the environment and speaking out on behalf of the L.G.B.T.Q. community.',
+};
+
+const speakers = [speaker1, speaker2, speaker3, speaker4, speaker5, speaker6];
+// Target open and close triggers in html
+
+const openEls = document.querySelectorAll('[data-open]');
+const closeEls = document.querySelectorAll('[data-close]');
+const openElems = Array.from(openEls);
+const closeElems = Array.from(closeEls);
+const isVisible = 'is-visible';
+const navLinks = document.querySelectorAll('.web-header-link');
+
+openElems.forEach((el) => {
+  el.addEventListener('click', function () {
+    const modalId = this.dataset.open;
+    document.getElementById(modalId).classList.add(isVisible);
+    document.body.style.overflow = 'hidden';
+  });
+});
+// Close modal
+closeElems.forEach((el) => {
+  el.addEventListener('click', function () {
+    this.parentElement.parentElement.classList.remove(isVisible);
+    document.body.style.overflow = 'auto';
+  });
+});
+
+// add event listener on click outside modal to close modal and enable scroll
+document.addEventListener('click', (e) => {
+  if (e.target === document.querySelector('.modal.is-visible')) {
+    document.querySelector('.modal.is-visible').classList.remove(isVisible);
+    document.body.style.overflow = 'auto';
+  }
+});
+
+// add event listener on esc key to close modal and enable scroll
+document.addEventListener('keyup', (e) => {
+  if (e.key === 'Escape' && document.querySelector('.modal.is-visible')) {
+    document.querySelector('.modal.is-visible').classList.remove(isVisible);
+    document.body.style.overflow = 'auto';
+  }
+});
+
+// add event listener on a tags to close modal, enable scroll, and go to section ID
+navLinks.forEach((elem) => {
+  elem.addEventListener('click', () => {
+    document.querySelector('.modal.is-visible').classList.remove(isVisible);
+    document.body.style.overflow = 'auto';
+  });
+});
+
+// Create Speaker Section HTML
+
+const speakerSection = document.querySelector('#speakers');
+// Create inner Div
+
+const innerDiv = document.createElement('div');
+innerDiv.className = 'inner-content';
+const speakerHeader = document.createElement('div');
+speakerHeader.className = 'about-section-header';
+const speakerTitle = document.createElement('h2');
```

Comment:
- Good job on building the section from Javascript. 👏

Reply:
Thanks 🙏